doc/cups.css

```diff
@@ -646,8 +646,8 @@ table.list tr:nth-child(even) {
 table.list th {
   border-right: 2px solid gray;
   font-family: monospace;
-  padding: 5px 10px 5px 2px;
-  text-align: right;
+  padding: 5px 10px 5px 12px;
```

Comment:
Is there a reason why you changed the padding?

Reply:
with 2px, like it was, it looked a little too sticky to the vertical line on the left. IMHO looks better this way.